modules/rook/examples/simple/main.tf

```diff
@@ -1,6 +1,6 @@
 # TEMPLATE: This file was automatically generated with `generate_addon_structure.sh`
 # TEMPLATE: and should be modified as necessary
-module "terraform-equinix-kubernetes-addons" {
+module "terraform_equinix_kubernetes_addons" {
```

Comment:
This is just an example so the name isn't that important, but I think we could remove the `terraform_` if nothing else.